sdk/containerservice/azure-mgmt-containerservice/azure/mgmt/containerservice/v2020_09_01/models/_models.py

```diff
@@ -0,0 +1,2577 @@
+        'windows_profile': {'key': 'properties.windowsProfile', 'type': 'ManagedClusterWindowsProfile'},
+        'service_principal_profile': {'key': 'properties.servicePrincipalProfile', 'type': 'ManagedClusterServicePrincipalProfile'},
+        'addon_profiles': {'key': 'properties.addonProfiles', 'type': '{ManagedClusterAddonProfile}'},
+        'node_resource_group': {'key': 'properties.nodeResourceGroup', 'type': 'str'},
+        'enable_rbac': {'key': 'properties.enableRBAC', 'type': 'bool'},
+        'enable_pod_security_policy': {'key': 'properties.enablePodSecurityPolicy', 'type': 'bool'},
+        'network_profile': {'key': 'properties.networkProfile', 'type': 'ContainerServiceNetworkProfile'},
+        'aad_profile': {'key': 'properties.aadProfile', 'type': 'ManagedClusterAADProfile'},
+        'auto_scaler_profile': {'key': 'properties.autoScalerProfile', 'type': 'ManagedClusterPropertiesAutoScalerProfile'},
+        'api_server_access_profile': {'key': 'properties.apiServerAccessProfile', 'type': 'ManagedClusterAPIServerAccessProfile'},
+        'disk_encryption_set_id': {'key': 'properties.diskEncryptionSetID', 'type': 'str'},
+        'identity_profile': {'key': 'properties.identityProfile', 'type': '{UserAssignedIdentity}'},
+    }
+
+    def __init__(
+        self,
+        **kwargs
+    ):
+        super(Components1Q1Og48SchemasManagedclusterAllof1, self).__init__(**kwargs)
+        self.identity = kwargs.get('identity', None)
+        self.provisioning_state = None
+        self.power_state = None
+        self.max_agent_pools = None
+        self.kubernetes_version = kwargs.get('kubernetes_version', None)
+        self.dns_prefix = kwargs.get('dns_prefix', None)
+        self.fqdn = None
+        self.private_fqdn = None
+        self.agent_pool_profiles = kwargs.get('agent_pool_profiles', None)
+        self.linux_profile = kwargs.get('linux_profile', None)
+        self.windows_profile = kwargs.get('windows_profile', None)
+        self.service_principal_profile = kwargs.get('service_principal_profile', None)
+        self.addon_profiles = kwargs.get('addon_profiles', None)
+        self.node_resource_group = kwargs.get('node_resource_group', None)
+        self.enable_rbac = kwargs.get('enable_rbac', None)
+        self.enable_pod_security_policy = kwargs.get('enable_pod_security_policy', None)
+        self.network_profile = kwargs.get('network_profile', None)
+        self.aad_profile = kwargs.get('aad_profile', None)
+        self.auto_scaler_profile = kwargs.get('auto_scaler_profile', None)
+        self.api_server_access_profile = kwargs.get('api_server_access_profile', None)
+        self.disk_encryption_set_id = kwargs.get('disk_encryption_set_id', None)
+        self.identity_profile = kwargs.get('identity_profile', None)
+
+
+class Components1Umhcm8SchemasManagedclusteridentityPropertiesUserassignedidentitiesAdditionalproperties(msrest.serialization.Model):
```

Comment:
This is the same issue in https://github.com/Azure/azure-sdk-for-python/issues/14150, consider to generate again after this PR merged: https://github.com/Azure/azure-rest-api-specs/pull/11190

CC to @jsntcy